test/run-local-e2e.sh

```diff
@@ -10,82 +10,115 @@ log_error() { echo -e "${RED}[ERROR]${NC} $*"; }
+    echo ""
+    echo "Arguments:"
+    echo "  test_type           Type of test to run"
+    echo "  kurtosis_repo_path  Path to Kurtosis CDK repo (use '-' to skip setup)"
+    echo "  e2e_repo_path       Path to E2E repo (use '-' to skip tests)"
+    echo ""
+    echo "Examples:"
+    echo "  $0 single-l2-network-fork12-op-succinct /path/to/kurtosis-repo /path/to/e2e-repo    # Run both setup and tests"
+    echo "  $0 single-l2-network-fork12-op-succinct /path/to/kurtosis-repo -                    # Run only setup"
+    echo "  $0 single-l2-network-fork12-op-succinct - /path/to/e2e-repo                         # Run only tests"
     exit 1
 fi
 
 TEST_TYPE=$1
-KURTOSIS_FOLDER=$2
-E2E_FOLDER=${3:-""}
+KURTOSIS_REPO_PATH=$2
+E2E_REPO_PATH=$3
 
 PROJECT_ROOT="$PWD"
 log_info "Starting local E2E setup..."
 
-if [ "$(docker images -q aggkit:local | wc -l)" -eq 0 ]; then
-    log_info "Building aggkit:local docker image..."
-    pushd "$PROJECT_ROOT" >/dev/null
-    make build-docker
-    make build-tools
-    chmod +x "./target/aggsender_find_imported_bridge"
-    popd >/dev/null
-else
-    log_info "Docker image aggkit:local already exists."
-fi
-
-log_info "Using provided Kurtosis CDK repo at: $KURTOSIS_FOLDER"
-
-pushd "$KURTOSIS_FOLDER" >/dev/null
-log_info "Cleaning any existing Kurtosis enclaves..."
-kurtosis clean --all
-
-log_info "Starting Kurtosis enclave"
+# Set ENCLAVE_NAME based on test type
 case "$TEST_TYPE" in
 single-l2-network-fork12-op-succinct)
     ENCLAVE_NAME="op"
-    kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_single_chain_fork12_op_succinct_args.json" .
     ;;
 single-l2-network-fork12-pessimistic)
     ENCLAVE_NAME="aggkit"
-    kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_single_chain_fork12_pessimistic_args.json" .
     ;;
 multi-l2-networks-2-chains)
     ENCLAVE_NAME="aggkit"
-    kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_multi_chains_args_1.json" .
-    kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_multi_chains_args_2.json" .
     ;;
 multi-l2-networks-3-chains)
     ENCLAVE_NAME="aggkit"
-    kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_multi_chains_args_3.json" .
-    kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_multi_chains_args_4.json" .
-    kurtosis run --enclave "$ENCLAVE_NAME" --args-file "$PROJECT_ROOT/.github/test_e2e_multi_chains_args_5.json" .
     ;;
 *)
     log_error "Unknown test type: $TEST_TYPE"
     exit 1
     ;;
 esac
-log_info "$ENCLAVE_NAME enclave started successfully."
-popd >/dev/null
 
-if [ -n "$E2E_FOLDER" ]; then
-    if [ ! -d "$E2E_FOLDER" ]; then
-        log_error "The provided E2E folder does not exist: $E2E_FOLDER"
+# Run Kurtosis setup if path is provided and not '-'
+if [ "$KURTOSIS_REPO_PATH" != "-" ]; then
```

Comment:
Should we check if the `KURTOSIS_REPO_PATH` is empty as well?

Reply:
But aren't we making them optional? To me it makes sense, if I do not define the `kurtosis_repo_path` flag, it will not run kurtosis. Its kind of easier than to have to put a `-` for that flag. Just thinking from a user perspective.

Same for `e2e_repo_path`.

Reply:
But, I see that you always expect 3 parameters. Its easier for the code probably. Which makes sense, since only we are going to use this. So, I am ok than to leave it as it is.